src/data/pokemon/item_effects.h

```diff
@@ -325,42 +325,66 @@ const u8 gItemEffect_DireHit[8] = {
 };
 
 const u8 gItemEffect_XAttack[8] = {
+#ifdef BATTLE_ENGINE
+    [1] = ITEM1_X_ATTACK,
+#else
     [0] = 1,
+#endif
     [5] = ITEM5_FRIENDSHIP_LOW | ITEM5_FRIENDSHIP_MID,
     [6] = 1,
     [7] = 1,
 };
 
 const u8 gItemEffect_XDefend[8] = {
+#ifdef BATTLE_ENGINE
+    [1] = ITEM1_X_DEFEND,
+#else
     [1] = 1 << 4,
+#endif
     [5] = ITEM5_FRIENDSHIP_LOW | ITEM5_FRIENDSHIP_MID,
     [6] = 1,
     [7] = 1,
 };
 
 const u8 gItemEffect_XSpeed[8] = {
+#ifdef BATTLE_ENGINE
+    [1] = ITEM1_X_SPEED,
+#else
     [1] = 1,
+#endif
     [5] = ITEM5_FRIENDSHIP_LOW | ITEM5_FRIENDSHIP_MID,
     [6] = 1,
     [7] = 1,
 };
 
 const u8 gItemEffect_XAccuracy[8] = {
+#ifdef BATTLE_ENGINE
+    [1] = ITEM1_X_ACCURACY,
+#else
     [2] = 1 << 4,
+#endif
     [5] = ITEM5_FRIENDSHIP_LOW | ITEM5_FRIENDSHIP_MID,
     [6] = 1,
     [7] = 1,
 };
 
 const u8 gItemEffect_XSpecialAttack[8] = {
+#ifdef BATTLE_ENGINE
+    [1] = ITEM1_X_SPATK,
+#else
     [2] = 1,
+#endif
     [5] = ITEM5_FRIENDSHIP_LOW | ITEM5_FRIENDSHIP_MID,
     [6] = 1,
     [7] = 1,
 };
 
 const u8 gItemEffect_XSpecialDefense[8] = {
+#ifdef BATTLE_ENGINE
+    [1] = ITEM1_X_SPDEF,
+#else
     [2] = 1,
+#endif
```

Comment:
```suggestion
#ifdef BATTLE_ENGINE
    [1] = ITEM1_X_SPDEF,
#endif
```
This change will prevent it from being used and gives the message that it won't have any effect. This way it only can be used when BE is merged in and not give the false impression that it actually works in IE by itself.